weave/trace_server/clickhouse_trace_server_batched.py

```diff
@@ -1397,7 +1406,7 @@ def _end_call_for_insert_to_ch_insertable_end_call(
         id=end_call.id,
         exception=end_call.exception,
         ended_at=end_call.ended_at,
-        summary_dump=_dict_value_to_dump(end_call.summary),
+        summary_dump=_any_value_to_dump(end_call.summary),
```

Comment:
this is not quite right. We want to ensure that `summary_dump` is in fact a dictionary dump. so `_dict_value_to_dump` is correct. I am not sure what is the motivation for this change as it opens us up to potentially non-dict data

Reply:
i think you want what was here before: `_dict_value_to_dump(dict(end_call.summary))`. 

I think the key here is that end_call.summary is now Expected to be a TypedDict after your typing changes.